src/overlays/actors/ovl_En_Sc_Ruppe/z_en_sc_ruppe.c

```diff
@@ -15,10 +15,16 @@ void EnScRuppe_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnScRuppe_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnScRuppe_Draw(Actor* thisx, GlobalContext* globalCtx);
 
+void func_80BD6910(EnScRuppe* this, GlobalContext* globalCtx);
+s32 func_80BD697C(s16 this);
```

Comment:
```suggestion
s32 func_80BD697C(s16 rupeeIndex);
```
Don't forget to update this as well.

Reply:
Not addressed yet?

Reply:
Yes, just forgot about the prototype, it wasn't needed anyway so these and other prototypes went bye bye